labman/db/tests/test_process.py

```diff
@@ -40,21 +40,11 @@ def _help_compare_timestamps(input_datetime):
     # is within 60 seconds of time at which process was created.
     # This is a heuristic--may fail if you e.g. put a breakpoint
     # between create call and assertLess call.
-    time_diff = datetime.now(timezone.utc) - input_datetime
+    time_diff = datetime.now() - input_datetime
     is_close = time_diff.total_seconds() < 60
     return is_close
 
 
-def _help_make_datetime(input_datetime_str):
-    # input_datetime_str should be in format '2017-10-25 19:10:25-0700'
-    return datetime.strptime(input_datetime_str, '%Y-%m-%d %H:%M:%S%z')
-
-
-def _help_format_datetime(input_datetime):
```

Comment:
I think we may want to keep this: it is much easier than writing out `datetime.strftime(input_datetime, Process.get_date_format())` each time we need to test a date, and using `str(input_datetime)` (while shorter) undesirably includes the seconds.

Reply:
I actually like writing the full thing, cause is a  single line (AKA IMOO not function worthy) and we are talking on saving chars vs. lines, and can yield the confusion that we had between _help_format_datetime vs _help_make_datetime ... but fine ...

Reply:
Ok; I'm willing to cede this one as a personal preference rather than a necessity :D